tests/test_parser.py

```diff
@@ -4,11 +4,14 @@
 THIS_FOLDER = os.path.dirname(os.path.abspath(__file__)) + "/.."
 sys.path.insert(1, THIS_FOLDER)
 
+from typing import List
+
 from vyxal.lexer import *
 from vyxal.parse import *
+from vyxal.structure import Structure
```

Comment:
looks like someone didn't properly test this last time..... :þ

we need to start actually using a type-checker (mypy?)

Reply:
Yeah, we really do. These tests still aren't working btw, but I'll try to get them working in a few days.

Reply:
This is something I've known about for a while - it's been something I've been thinking about since I made the structure classes.